tools/test-lb-service-catalyst-cloud.sh

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env bash
```

Comment:
this is only used for CPO job, no need to add it here. Just move to the job folder.

Reply:
OK